lib/experimental/Widgets/Content/CopyLabel/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Icon } from "@/components/Utilities/Icon"
+import { LayersFront } from "@/icons"
+import { forwardRef } from "react"
+interface CopyLabelType {
+  text: string
```

Comment:
We could accept arbitrary elements as `children` but I'm afraid of abuse. I think we can limit this to a `label` property or `text` as you have and maybe later open it to other use cases.